integration-tests/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -60,6 +61,9 @@
  */
 public class TfsShellTest {
   private static final int SIZE_BYTES = Constants.MB * 10;
+  @Rule
+  public LocalTachyonClusterResource mLocalTachyonClusterResource =
+      new LocalTachyonClusterResource(SIZE_BYTES, 1000, Constants.GB);
   private LocalTachyonCluster mLocalTachyonCluster = null;
```

Comment:
Yeah I left it in there because a lot of methods call methods on the `mLocalTachyonCluster`, so rather than modify all of them, I just made it a local variable. Only the `LocalTachyonClusterResource` actually manages a cluster, though.

On a side note, I noticed there was some explicit cluster configuration that can be moved to the cluster resource constructor, so I'll change that.